HUD/EndTextCommandIsThisHelpMessageBeingDisplayed.md

```diff
@@ -11,7 +11,15 @@ BOOL END_TEXT_COMMAND_IS_THIS_HELP_MESSAGE_BEING_DISPLAYED(int hudIndex);
 
 
 ## Parameters
-* **hudIndex**: The hud index for help message. `0` is `HELP_TEXT`, `1` and `2` is `FLOATING_HELP_TEXT_1` and `FLOATING_HELP_TEXT_2`.
+* **hudIndex**: The hud index for help message.
+```csharp
```

Comment:
Move all this C# code over to line 12, I attached a file showing you how it should look, you can replace everything with the contents from that file I attached if you want.

[EndTextCommandIsThisHelpMessageBeingDsiplayed.md](https://github.com/citizenfx/natives/files/11817954/EndTextCommandIsThisHelpMessageBeingDsiplayed.md)

Reply:
Thank you, I have replaced the content you sent to me and made a commit.

Reply:
> Thank you, I have replaced the content you sent to me and made a commit.

Amazing, thank you! I'll approve this now 🙂